core/src/main/java/nl/nn/adapterframework/ftp/FtpSession.java

```diff
@@ -68,6 +68,7 @@
  */
 public class FtpSession {
 	protected Logger log = LogUtil.getLogger(this);
+	private ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
```

Comment:
This class already has a configure method, If you add the `IConfigurable` interface here you don't have to add it on it's children.